### DISCUSSION TITLE
Error when setting up a new course

### DISCUSSION THREAD

**Discussion by travislatchman** [inducer/relate#945] [Q&A]
I encountered the following error when attempting to set up a new course. I am not sure what the source of this error could be and what it means. Appreciate any feedback and guidance. The code was only modified to allow for the ability to jump from one flow to another flow.

![image](https://user-images.githubusercontent.com/32372013/182612189-5524afab-fee6-4bbe-b003-5a045ed62c3d.png)

**Comment by inducer** [comment]
What was the general form of GIT URL you used?

It might be useful to turn on debug mode (in `local_settings.py`), and to defeat this exception handler:

https://github.com/inducer/relate/blob/721f2a65d9a9970e3fd4a527c0eb4540ea7e4afa/course/versioning.py#L277-L290

 to get a more complete backtrace and post that here.
